CHANGELOG.md

```diff
@@ -109,15 +125,17 @@ Adding a new version? You'll need three changes:
 - Removed support for deprecated `KongIngress` fields: `Proxy` and `Route`. Respective
   `Service` or `Ingress` annotations should be used instead. See [KIC Annotations reference].
   [#4760](https://github.com/Kong/kubernetes-ingress-controller/pull/4760)
-- Removed previously deprecated flags:
+- Removed support for CLI flags: 
   - `sync-rate-limit`: Use `--proxy-sync-seconds` instead
   - `stderrthreshold`: Has no effect
```

Comment:
IDK if "Has no effect" is valid description, because it is removed entirely thus usage results in error, and the controller won't start. So maybe just group one under the other without any annotation, because it's in the header "Removed previously deprecated flags" or you have different idea

Reply:
👍 I removed all the descriptions beside removed flags, because they were not very useful in my opinion.